enterprise_access/apps/api_client/lms_client.py

```diff
@@ -153,6 +154,40 @@ def get_enterprise_admin_users(self, enterprise_customer_uuid):
 
         return results
 
+    def create_enterprise_admin_user(self, enterprise_customer_uuid, user_email):
```

Comment:
[sanity check / consideration] Will we have a need to create multiple enterprise admin users at once during provisioning? Or will we be able to sufficiently get by with only creating 1 enterprise admin user at a time, as suggested here?

Adding support for the multiple known admin users use case is definitely something that could (and probably should) be deferred, though, with the assumption being additional admins could be added self-serve within the Admin Portal in the future.

Just wanted to verify we're confident we don't need to support the multiple enterprise admin users during provisioning use case to start! :)

Reply:
I had the same thought - I think we can assume only one for now, and work out details on supporting multiple later. AFAICT, the enterprise pending admin endpoint doesn't support adding multiple admins in a single request.

Reply:
> AFAICT, the enterprise pending admin endpoint doesn't support adding multiple admins in a single request.

That's my understanding, as well. More reason to deliberately defer on supporting multiple enterprise admins for now.